### PR TITLE
improved navbar padding, view, title of page

### PR DESCRIPTION
**Pull request by Rupeshiya**
**Removed padding from navbar**
nav
**Removed overflow from navbar**
mobile

**Updated title of the page**

### CODE REVIEW

**Comment by Rupeshiya** [comment]
@ayushnagar123 @jaskirat2000 please review

**Comment by jaskirat2000** [comment]
Hi Rupesh, Desktop looks good but Mobile view doesn't look that good. Can
you try something other for navbar of mobile view ?

**Comment by Rupeshiya** [comment]
sure @jaskirat2000 , can you please provide some example, how it should look like?

File: src/App.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ function App() {
               <Link to="/joinus"><Nav.Link href="/joinus">Join Us</Nav.Link></Link>
             </Nav>
           </Navbar.Collapse>
-        </Container>
+        </div>
```
Comment (vaibhavdaren) [inline src/App.js]: /container should be there i guess.
have we replaced this with nav divs?

**Comment by jaskirat2000** [comment]
Sure check out some examples here https://images.app.goo.gl/HkBYNZqzjgF9tbnn7

File: src/App.js (hunk 1 above)
Comment (Rupeshiya) [reply src/App.js]: yeah, It has been replaced by div,

**Pull request merged**